test/moves/toxic-spikes.test.ts

```diff
@@ -112,7 +112,7 @@ describe("Moves - Toxic Spikes", () => {
     game.move.select(MoveId.SPLASH, 1);
     await game.toNextTurn();
 
-    const arenaTags = game.scene.arena.getTagOnSide(ArenaTagType.TOXIC_SPIKES, ArenaTagSide.ENEMY) as EntryHazardTag;
+    const arenaTags = game.scene.arena.findTag(ArenaTagType.TOXIC_SPIKES, ArenaTagSide.ENEMY) as EntryHazardTag;
     expect(arenaTags.tagType).toBe(ArenaTagType.TOXIC_SPIKES);
     expect(arenaTags.layers).toBe(1);
```

Comment:
not sure that's the best way to handle the undefined case
```suggestion
    const arenaTag = game.scene.arena.findTag<EntryHazardTag>(ArenaTagType.TOXIC_SPIKES, ArenaTagSide.ENEMY);
    expect(arenaTag).toBeDefined();
    expect(arenaTag!.tagType).toBe(ArenaTagType.TOXIC_SPIKES);
    expect(arenaTag!.layers).toBe(1);
```